ansible/playbooks/roles/redpanda_broker/templates/configs/tiered_storage.j2

```diff
@@ -0,0 +1,10 @@
+cluster:
+  cloud_storage_access_key: THISVALUENOTUSED
+  cloud_storage_bucket: {{ tiered_storage_bucket_name if tiered_storage_bucket_name is defined }}
+  cloud_storage_enable_remote_read: true
+  cloud_storage_enable_remote_write: true
```

Comment:
This makes all topics tiered-storage-enabled by default if the `tiered_storage_enabled` terraform variable is enabled.